pages/search/views/index.jsx

```diff
@@ -58,16 +58,15 @@ export default function Index () {
               prop="establishments"
               label={<Snippet>filter.label</Snippet>}
               options={['unassociated']}
-              showAllLabel="Show all"
+              showAll={{ position: 'before', label: 'Show all' }}
               formatter={uppercaseFirst}
             />
           }
           {
             hasFilters && <LinkFilter
               prop="status"
               label="Filter by status:"
-              showAllLabel="Show all"
-              showAllBefore={false}
+              showAll={{ position: 'after', label: 'Show all' }}
```

Comment:
Is it correct that for two adjacent sets of filters we have the "Show all" link in different places? Seems weird.

Reply:
Will merge this and ask questions later.